noxfile.py

```diff
@@ -20,10 +20,40 @@
     "jax>=0.4.32",
     "pytest",
     "pytest-benchmark",
+    "coverage[toml]",
 ]
 GLASS_REPO_URL = "https://github.com/glass-dev/glass"
 
 
+def _get_revisions_from_posargs(
+    posargs: list[str],
+    expected_num_revisions: int,
+) -> tuple[str, str]:
+    if not posargs:
+        msg = "Revision not provided"
+        raise ValueError(msg)
+
+    n = len(posargs)
+
+    if n == 1 and expected_num_revisions in {1, 2}:
+        return posargs[0], ""
+    if n == expected_num_revisions == 2:  # noqa: PLR2004
```

Comment:
Would be much more readable

```suggestion
    if n in {expected_num_revisions, 2}:  # noqa: PLR2004
```

Reply:
```suggestion
    if n == expected_num_revisions and n == 2:  # noqa: PLR2004
```